core/runtime/setup.rs

```diff
@@ -23,13 +23,11 @@ fn v8_init(
 
   let base_flags = concat!(
     " --wasm-test-streaming",
-    " --harmony-import-assertions",
-    " --harmony-import-attributes",
     " --no-validate-asm",
     " --turbo_fast_api_calls",
-    " --harmony-array-from_async",
-    " --harmony-iterator-helpers",
     " --harmony-temporal",
+    " --js-explicit-resource-management",
+    " --js-float16array",
```

Comment:
Can you add a small test to `testing/unit/tc39_test.ts` that verifies `Float16Array` is present?